README.md

```diff
@@ -1,4 +1,32 @@
 # delivery-base
```

Comment:
Make sure to include a supermarket cookbook version badge if this is going to get released on Supermarket